v2/cli/init_command.go

```diff
@@ -0,0 +1,243 @@
+		GitCloneURL: c.String(flagGit),
+	}
+
+	err := core.ValidateProjectID(fields.ProjectID)
+	if err != nil {
+		return err
+	}
+
+	// Check if input language is valid
+	fields.Language, err = canonicalizeLanguage(fields.Language)
+	if err != nil {
+		return err
+	}
+
+	// Check if the .brigade directory exists
+	err = file.EnsureDirectory(".brigade")
+	if err != nil {
+		return err
+	}
+
+	// Generate js/ts buffer
+	var scriptBytes []byte
+	scriptBytes, err = execTemplate(scriptTemplate, fields)
+	if err != nil {
+		return err
+	}
+	fields.Script = string(scriptBytes)
+
+	brigadeScriptFileName := fmt.Sprintf("brigade.%s", fields.Language)
+
+	// If git repo is provided, create a separate js/ts file for script
+	var overwrite bool
+	if fields.GitCloneURL != "" {
+		scriptPath := path.Join(".brigade", brigadeScriptFileName)
+		if overwrite, err = checkFileOverwrite(scriptPath); err != nil {
+			return err
+		} else if overwrite {
+			if err = ioutil.WriteFile(
+				scriptPath,
+				scriptBytes,
+				0644,
+			); err != nil {
+				return err
+			}
+		}
+	}
+
+	// Generate the project.yaml file
+	var projectBytes []byte
+	projectBytes, err = execTemplate(projectTemplate, fields)
+	if err != nil {
+		return err
+	}
+
+	// Check if the .brigade/project.yaml path exists.
+	// Confirm overwrite if so.
+	projectPath := path.Join(".brigade", "project.yaml")
+	if overwrite, err = checkFileOverwrite(projectPath); err != nil {
+		return err
+	} else if overwrite {
+		if err = ioutil.WriteFile(
+			projectPath,
+			projectBytes,
+			0644,
+		); err != nil {
+			return err
+		}
+	}
+
+	var notesBytes []byte
+	notesBytes, err = execTemplate(notesTemplate, fields)
+	if err != nil {
+		return err
+	}
+
+	// Create a NOTES.txt file
+	notesPath := path.Join(".brigade", "NOTES.txt")
+	if overwrite, err = checkFileOverwrite(notesPath); err != nil {
+		return err
+	} else if overwrite {
+		if err = ioutil.WriteFile(
+			notesPath,
+			notesBytes,
+			0644,
+		); err != nil {
+			return err
+		}
+	}
+
+	var secretsBytes []byte
+	secretsBytes, err = execTemplate(secretsTemplate, fields)
+	if err != nil {
+		return err
+	}
+
+	// Create a secrets.yaml file
+	secretsPath := path.Join(".brigade", "secrets.yaml")
+	if overwrite, err = checkFileOverwrite(secretsPath); err != nil {
+		return err
+	} else if overwrite {
+		if err = ioutil.WriteFile(
+			secretsPath,
+			secretsBytes,
+			0644,
+		); err != nil {
+			return err
+		}
+	}
+
+	err = addToGitignore(".brigade/secrets.yaml", "./.gitignore")
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Check if the dirPath directory exists. If not, create it
+// func ensureDirectory(dirPath string) error {
+// 	if dirExists, err := file.Exists(dirPath); err != nil {
+// 		return err
+// 	} else if !dirExists {
+// 		fmt.Println("Creating " + dirPath + " directory...")
+// 		err = os.Mkdir(dirPath, 0755)
+// 		if err != nil {
+// 			return err
+// 		}
+// 	}
+// 	return nil
+// }
```

Comment:
Let's remove this commented code.

Reply:
Wow I can't believe I forgot to remove that after testing, I'll be more careful next time!

Reply:
No worries. Fixed easily enough.